spec/bracket-matcher-spec.coffee

```diff
@@ -544,6 +544,44 @@ describe "bracket matching", ->
         expect(buffer.lineForRow(0)).toBe "{}"
         expect(editor.getCursorBufferPosition()).toEqual([0, 1])
 
+    describe "when addPairs configuration is set globally", ->
+      it "inserts a matching carat", ->
+        atom.config.set 'bracket-matcher.addPairs', ['<:>']
+        editor.setCursorBufferPosition([0, 0])
+        editor.insertText '<'
+        expect(buffer.lineForRow(0)).toBe "<>"
+        expect(editor.getCursorBufferPosition()).toEqual([0, 1])
+
+    # Scope tests inexplicably fail
```

Comment:
I started down a scary road on this one - ended up time-boxing and disabling the tests at this point. The problem seems to be with the Spec library itself in the mocking of the testing editor. I added a bunch of console.log lines all over the code - all of them confirmed that I was grabbing from the correct scope. For some reason, the way the test codebase is implementing the atom.config.set at scope level, it isn't triggering a callback.

I gave myself a TODO to create some tests of the tests (heh) to try to consistently recreate what I see as a potential bug. Then I can go about fixing.